metricbeat/module/prometheus/collector/config.go

```diff
@@ -0,0 +1,38 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package collector
+
+type metricsetConfig struct {
+	MetricsFilters MetricFilters `config:"metrics_filters"`
+}
+
+type MetricFilters struct {
+	IncludeMetrics *[]string `config:"include_metrics"`
+	IgnoreMetrics  *[]string `config:"ignore_metrics"`
```

Comment:
Umm, sorry, I hadn't thought about this config structs. With current implementation it seems that the options will be named `metrics_filters.include_metrics` and `metrics_filters.ignore_metrics`. `metrics` can be a bit redundant there. Wdyt about reducing them to something like `metrics_filters.include` and `metrics_filters.ignore`?
```suggestion
	IgnoreMetrics  *[]string `config:"ignore"`
```

Reply:
I will go along!